nessie/src/test/java/org/apache/iceberg/nessie/BaseTestIceberg.java

```diff
@@ -114,8 +113,8 @@ private void resetData() throws NessieConflictException, NessieNotFoundException
   }
 
   @BeforeEach
-  public void beforeEach(@NessieUri URI nessieUri) throws IOException {
-    this.uri = nessieUri.toString();
+  public void beforeEach(@NessieClientUri URI nessieUri) throws IOException {
+    this.uri = nessieUri.resolve("v1").toASCIIString();
```

Comment:
With `@NessieClientUri` it is preferable to also use `@NessieApiVersions` (cf. examples in the Nessie repo). This way, the test framework will inject the correct URI for the API version.

Reply:
Thanks for the review. I have updated the code to use `NessieApiVersions` now